### ISSUE TITLE
Review translated text that is concatinated with variables

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#840]
## TL;DR 

Verify all translated text is translated in whole string and not as concatenated parts around variables. 

## Narrative
Translating text is not an easy task.  To facilitate the translators work we should appropriately format strings. 

**Examples of BAD string formatting**
```
_('asdf ') + some_variable + _(' qwer')
```

**Examples of GOOD string formatting**
```
_(f'asdf {some_varialble} qwer')
```

I am sure there are other examples, but they escape me at the moment. 

## Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests
- [ ] Alternatives Considered

## Implementation Notes
We should refer to MBAH for any questions on best practices.

**Comment by okaycj** [comment]
This is not necessary.  It would be nice to clean up the code, but there a lot to do around here.  Closing.
